bin/quality_controls.qmd

```diff
@@ -8,57 +8,88 @@ jupyter: python3
 
 # Introduction
 
+<!-- TODO: Add SpatialData in intro -->
```

Comment:
This still needs to be done, @cavenel .

Reply:
I think it is good to keep the intro as is: (`The AnnData format is utilized to organize and store the Spatial Transcriptomics data. It includes [...]. The AnnData object is saved in a zarr [SpatialData object](link)`.

SpatialData says in the doc:
> **Table (table of annotations for regions)**
> Annotations of regions of interest. Each row in this table corresponds to a single region on the coordinate space. This is represented as an **AnnData object** to allow for complex annotations on the data.

If that's fine with you, I'll remove the TODO comment.

Reply:
Okay, then let's keep it as-is.